arch/x86/kvm/vmx/pkvm/hyp/pci.c

```diff
@@ -52,6 +52,47 @@ static int pci_mmcfg_write(u64 address, int size, unsigned long value)
 	return 0;
 }
 
+unsigned long pkvm_pci_cfg_space_read(u32 bdf, u32 offset, int size)
+{
+	union pci_cfg_addr_reg reg;
+	unsigned long value = 0;
+
+	reg.enable = 1;
+	reg.bdf = bdf;
+	reg.reg = offset & (~0x3);
+
+	pci_cfg_space_read(&reg, offset & 0x3, size, &value);
+
+	return value;
+}
+
+void pkvm_pci_cfg_space_write(u32 bdf, u32 offset, int size, unsigned long value)
+{
+	union pci_cfg_addr_reg reg;
+
+	reg.enable = 1;
+	reg.bdf = bdf;
+	reg.reg = offset & (~0x3);
+
+	pci_cfg_space_write(&reg, offset & 0x3, size, value);
+}
+
+static bool host_vpci_cfg_space_audit_access(struct pkvm_ptdev *ptdev, u64 offset, int size, u32 value)
```

Comment:
prefer to use name host_vpci_cfg_data_allow_write.

Reply:
Changed the name.